docs/uma-tokenholders/guidence-on-collateral-currency-addition.md

```diff
@@ -0,0 +1,44 @@
+---
+title: Guidance for the Addition of Collateral Currencies
+sidebar_label: Guidance for the Addition of Collateral Currencies
+---
+Collateral currencies can be added through the collateral currency UMIP template available on Github, by adding 
+
+ - the name of the token to the be added 
+ - the contract address of the token  together with a link to the address on etherscan
+ - the amount that should be added to the store as final fee expressed in units of the intended collateral currency.   
+_This should equate to approximately $400 at the time the UMIP is submitted.  If the token is not directly listed on Coingecko and/or Coinmarketcap, please add how the estimated dollar value of the token was obtained in the Rationale section._
+
+***Note**: rebasing tokens cannot be added as collateral currencies as they do not work with UMA's factory contracts*
+
+____________
+The below provides some guidelines when using collateral currencies in an UMA contract
+
+**Stablecoins**
```

Comment:
I would make these headings instead of bolded text

Reply:
Have changed